File: inflearn/algorithm/1-02/main.js
const ps = require('../modules/ps');

const solution = (input) => {
  let answer = '';
  for (let i = 0; i < input.length; i++) {
    if (input[i] >= 'a') answer += input[i].toUpperCase()
    else answer += input[i].toLowerCase();
  }
  return answer;
};

ps.main(async () => {
  let input = await ps.readLine(); // 문자열 입력

  console.log(solution(input));
});

File: inflearn/algorithm/1-02/main.dart
import 'dart:io';

String solution(String str) {
  String answer = '';
  var check = 'abcdefghijklmnopqrstuvwxyz';
  for (var s in str.split('')) {
    if (check.contains(s)) {
      answer += s.toUpperCase();
    } else {
      answer += s.toLowerCase();
    }
  }
  return answer;
}

void main() {
  String input = stdin.readLineSync()!; // 문자열 입력

  print(solution(input));
}
